### PR TITLE
[Ready] The CE can now command Poly to shut it, or perch on their shoulder.

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request
Ready and tested locally.

image

She will only ever listen to the CE.

## Why It's Good For The Game

Adds some basic voice command recognition to best bird. Squawk!

## Changelog
:cl:
add: Poly now listens to basic voice commands, but she only recognizes the CE's authority!
add: There's three keywords: Perch, hop (off), shut (silences for a while). You need to adress the bird for her to listen.
add: Example command sentences: "Poly, perch on me.", "Poly, shut." "Poly, hop off." "Get on my shoulder, Poly."
/:cl:

### CODE REVIEW

**Comment by Strazyplus** [comment]
merge this immediately telling birds to shut up is great!

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -909,6 +916,64 @@
...
+		buckled.unbuckle_mob(src, TRUE)
+		emote("me", EMOTE_VISIBLE, "squawks and hops off of [buckled], flying away.")
+	buckled = null
+	buckled_to_human = FALSE
+	pixel_x = initial(pixel_x)
+	pixel_y = initial(pixel_y)
+
+/mob/living/simple_animal/parrot/Poly/proc/command_resume_speech(var/initial_chance) // Skyrat proc
+	speak_chance = initial_chance
+
```
Comment (jjpark-kb) [inline code/modules/mob/living/simple_animal/parrot.dm:976]: all of this can be modular

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 2]
```diff
@@ -148,8 +149,13 @@
 		stat("Held Item", held_item)
 		stat("Mode",a_intent)
 
+/mob/living/simple_animal/parrot/proc/check_command() // Skyrat - Poly listens to some of the CE's commands!
+	return FALSE // Simply return false for non-Poly parrots
```
Comment (jjpark-kb) [inline code/modules/mob/living/simple_animal/parrot.dm:153]: this could be modular

Comment (NotRanged) [reply code/modules/mob/living/simple_animal/parrot.dm]: weh

**Comment by NotRanged** [comment]
Has been made modular now

**Pull request merged**